### PR TITLE
add accessPackageResourceRequest

### PR DESCRIPTION
**Pull request by markwahl-msft**
Introduce to entitlement management in Microsoft Graph beta more operations to manage resource and their roles relationships to catalogs and access packages.

### CODE REVIEW

File: api-reference/beta/api/accesspackage-post-accesspackageresourcerolescopes.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+### Request
+
+The following is an example of the request.
+<!-- {
+  "blockType": "request",
+  "name": "create_accesspackageresourcerolescope_from_accesspackage"
+}-->
+
+```http
+POST https://graph.microsoft.com/beta//identityGovernance/entitlementManagement/accessPackages/{id}/accessPackageResourceRoleScopes
```
Comment (dkershaw10) [inline api-reference/beta/api/accesspackage-post-accesspackageresourcerolescopes.md]: Spurious "/"
https://graph.microsoft.com/beta//identityGovernance -> https://graph.microsoft.com/beta/identityGovernance

File: api-reference/beta/api/accesspackage-post-accesspackageresourcerolescopes.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+<!-- {
+  "blockType": "response",
+  "truncated": true,
+  "@odata.type": "microsoft.graph.accessPackageResourceRoleScope"
+} -->
+
+```http
+HTTP/1.1 201 Created
+Content-type: application/json
+
```
Comment (dkershaw10) [inline api-reference/beta/api/accesspackage-post-accesspackageresourcerolescopes.md]: There should be a response here.  It should not be empty.

File: api-reference/beta/resources/accesspackageresourcerequest.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+## Properties
+
+| Property     | Type        | Description |
+|:-------------|:------------|:------------|
+|catalogId|String|The unique ID of the access package catalog.|
+|expirationDateTime|DateTimeOffset|The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 would look like this: `'2014-01-01T00:00:00Z'`|
+|id|String| Read-only.|
+|isValidationOnly|Boolean|If set, does not add the resource.|
+|justification|String|The requestor's justification for adding the resource.|
+|requestState|String| Read-Only.|
```
Comment (dkershaw10) [inline api-reference/beta/resources/accesspackageresourcerequest.md]: Fine as is, but is there any more info you should be adding here?  What values could a developer expect here?

File: api-reference/beta/resources/accesspackageresourcerequest.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+|requestStatus|String|Read-only.|
+|requestType|String|Use "AdminAdd" to add a resource.|
```
Comment (dkershaw10) [inline api-reference/beta/resources/accesspackageresourcerequest.md]: Is this the only supported type?

File: api-reference/beta/api/accesspackage-post-accesspackageresourcerolescopes.md (hunk 2 above)
Comment (markwahl-msft) [reply api-reference/beta/api/accesspackage-post-accesspackageresourcerolescopes.md]: (I agree, we're working on it)

**Pull request closed without merging**